README.md

```diff
@@ -1,4 +1,31 @@
 # Plio Docs
-This repo contains Plio's documentation (served as a static site on S3)
 
-You can access the docs from here
\ No newline at end of file
+This repository contains the documentation and guides for [Plio](https://plio.in)
+
+## Contributing
+This webapp is built using [VuePress](https://v2.vuepress.vuejs.org/)
+
+### Serve locally
+- Clone the repo and run `npm install`.
+- Run `npm run docs:dev` to serve locally.
+
+### Build static files for deployment
+- Run `npm run docs:build` to build the app's static files which can be deployed as a static web app.
```

Comment:
same as above

also, good to mention that running this command is not a necessary step for deployment.

```suggestion
- Run `npm run build` to build the app's static files which can be deployed as a static web app.
```